StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PrepaneViews.swift

```diff
@@ -0,0 +1,261 @@
+//
+//  PrepaneViewss.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 2/6/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+// A container that encapsulates all the subviews necessary
+// to create a prepane view. Helps to avoid bloat in
+// `PartnerAuthViewController`.
+final class PrepaneViews {
```

Comment:
This code is a copy-paste of `PrepaneView` except it removes the `UIView` subclass, and changes the loading state.

It's a bit unconventional to have a class just for storing references to views, but it serves its purpose to avoid more code bloat in `PartnerAuthViewController` while also leaving the code essentially the same as before. It's likely it will change, but for now I'm trying to keep code changes to a minimum because I am still learning how exactly the code will change shape as we transfer this "partner auth pane" to be rendered as a sheet (instead of full screen). Long story short, it will be a bit more than just changing the presentation logic (some API calls will be moved to other VC's too, error UI code for the whole auth flow might completely change).